SonarQube.Common/ProjectInfo/ProjectInfo.cs

```diff
@@ -11,14 +11,6 @@
 
 namespace SonarQube.Common
 {
-    public enum ProjectLanguage
```

Comment:
The project language is written to the ProjectInfo.xml file so we can't use an enum if we want to support unknown languages.